frontend/src/components/pages/VolunteerDashboard/ShiftCard.tsx

```diff
@@ -0,0 +1,129 @@
+import { Box, Button, Stack, Text } from "@chakra-ui/react";
+import { format, parse } from "date-fns";
+import React, { useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import DonorAPIClient from "../../../APIClients/DonorAPIClient";
+import * as Routes from "../../../constants/Routes";
+import {
+  CheckInWithShiftType,
+  ScheduleWithShiftType,
+  ShiftType,
+} from "../../../types/VolunteerTypes";
+import CardField from "../../common/CardField";
+import { getShiftColor } from "./types";
+
+interface CheckInOrScheduleProps {
+  donorId?: string;
+  isPickup?: boolean;
+  pickupLocation?: string;
+  volunteerTime?: string;
+  startTime?: string;
+  startDate?: string;
+  notes: string;
+  type: ShiftType;
+}
+const VolunteerShiftCard = ({
+  shift,
+}: {
+  shift: CheckInWithShiftType | ScheduleWithShiftType;
+}): JSX.Element => {
+  const {
+    donorId,
+    isPickup,
+    pickupLocation,
+    volunteerTime,
+    startTime,
+    startDate,
+    notes,
+    type,
+  } = shift as CheckInOrScheduleProps;
+  const [businessName, setBusinessName] = useState<string>("");
+
+  const history = useHistory();
+
+  const dateLocal = () => {
+    if (startDate) {
+      return format(new Date(startDate), "EEE MMM dd, yyyy");
+    }
+    return startTime && format(new Date(startTime), "EEE MMM dd, yyyy");
+  };
+
+  const timeLocal = () => {
+    if (startDate) {
+      return format(new Date(startDate), "h:mma");
+    }
+
+    return (
+      volunteerTime &&
+      format(parse(volunteerTime, "kk:mm", new Date()), "h:mma")
```

Comment:
I'm getting a parsing error here, is `volunteerTime` always that format?
When I logged on chrome I got:
![image](https://user-images.githubusercontent.com/19617248/161185915-67cba2d8-6ad3-4d92-96c0-90a3964cf8eb.png)

Reply:
Should `2021-11-06 19:25:32.806 -04:00` return `7:25 PM` (EDT)?